### PR TITLE
Add support of dumping in pcap format to dump.go example

### PR DESCRIPTION
**Pull request by mbulatova**

### CODE REVIEW

**Comment by ifilippov** [comment]
Maybe two options to one with non-bool parameter? Other great!

**Comment by ifilippov** [comment]
Also what about SetWriter instead of SetReceiver? It won't be cloned and it is kind of OK?

**Comment by mbulatova** [comment]
Yes SetWriter is unclonable like SetReceiver is. SetWriter is bound to fixed filename and is not supposed to clone. 
Here is just demonstration how pcap dumping can be used as clonable handler

File: examples/dump.go [hunk 1]
```diff
@@ -35,16 +35,23 @@ func main() {
 	secondFlow := flow.SetPartitioner(firstFlow, 50000000, 1)
 
 	// Dump separated packet. By default function dumper() is used.
-	if *hexdumpOn {
-		flow.SetHandler(secondFlow, hexdumper, nil)
-	} else {
+	switch *dumptype {
+	case 1:
```
Comment (ifilippov) [inline examples/dump.go]: 1 for hex, not?

File: examples/dump.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ var (
 )
 
 func main() {
-	hexdumpOn := flag.Bool("hex", false, "enable dumping of packets in hex format")
+	dumptype := flag.Uint("dumptype", 0, "dumping format type (0 - dumper function, 1 - hex, 2 - pcap file)")
```
Comment (ifilippov) [inline examples/dump.go]: synchronize this with switch, please

**Comment by ifilippov** [comment]
After correction of option handling you can merge - approved.

**Pull request merged**